src/errors/test-run/index.js

```diff
@@ -323,3 +323,30 @@ export class CurrentIframeIsInvisibleError extends TestRunErrorBase {
         super(TYPE.currentIframeIsInvisibleError);
     }
 }
+
+// Native dialog errors
+export class NativeDialogHandlerNotSetError extends TestRunErrorBase {
```

Comment:
mmm... Unhandled dialog or unhandled error?
`NativeDialogHandlerNotSetError` is less descriptive (doesn't imply that dialog invocation actually triggered the error) but less ambiguous...

what can I come up with...
`NativeDialogNotHandledError` may be...